### DISCUSSION TITLE
FILL_RECT works incorrectly in Amazfit Active Simulator

### DISCUSSION THREAD

**Discussion by ga2mer** [zepp-health/zeppos-forum#362] [Developer Feedback]
`FILL_RECT` does not draw more than 100 pixels in width and height on Amazfit Active image in simulator
<details>
  <summary>Amazfit Active on simulator</summary>

 ![Screenshot_20240809_043113](https://github.com/user-attachments/assets/c8f9f41c-00a0-4a8b-8de5-bc29139590a6)
  
</details>
<details>
  <summary>Amazfit Balance on simulator</summary>

![Screenshot_20240809_043048](https://github.com/user-attachments/assets/1e7a555c-b917-44e7-a599-102db1bf8903)
  
</details>

Also works correctly on real Amazfit Active hardware

OS: Arch Linux (6.10.3-arch1-2)
Simulator version: 2.0.0
Amazfit Active image version: v2.0.1

Sample widget creation code:
```
hmUI.createWidget(hmUI.widget.FILL_RECT, {
      x: 150,
      y: 150,
      w: 200,
      h: 100,
      color: 0xffffff,
});
```

**Comment by silver-zepp** [comment]
Hey, this is a known issue. However, due to architectural changes in the firmware to cater new simulator approach, this change is delayed.
